### PR TITLE
Issue 70

### PR DESCRIPTION
**Pull request by Kaliahh**

### CODE REVIEW

File: Client/Exceptions/CompletionException.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Client.Exceptions
+{
+    public class CompletionException : Exception
+    {
+        public CompletionException(string message) : base(message)
+        {
+
```
Comment (naitoarekkus) [inline Client/Exceptions/CompletionException.cs:9]:
```suggestion
        // Intentional left empty...
```
Bare for at vise at det er med vilje

**Review by zaztz — DISMISSED** [review]
Code Review:

- [x] The code solves the associated issue and has been sufficiently tested
- [x] The code makes sense and has been documented using comments or well named variables/functions
- [x] The code conforms to the convention

**Review by naitoarekkus — COMMENTED** [review]
There is my problem with "var" but its not that bad.
I have made a single suggestion to describe that it is intensional

**Review by naitoarekkus — DISMISSED** [review]
Approved

File: Client-app unit tests/Client-app unit tests.csproj [hunk 2]
```diff
@@ -8,6 +8,7 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="IronPython" Version="2.7.11" />
```
Comment (dkalaxdk) [inline Client-app unit tests/Client-app unit tests.csproj:11]:
```suggestion

```

File: Client/Client.csproj [hunk 3]
```diff
@@ -6,6 +6,7 @@
     </PropertyGroup>
 
     <ItemGroup>
+      <PackageReference Include="IronPython" Version="2.7.11" />
```
Comment (dkalaxdk) [inline Client/Client.csproj:9]:
```suggestion
```

**Review by dkalaxdk — CHANGES_REQUESTED** [review]
Changed noted in the review. But i had to make a comment ¯\_( ͡° ͜ʖ ͡°)_/¯

**Review by dkalaxdk — APPROVED** [review]

**Review by naitoarekkus — APPROVED** [review]
Approved

**Pull request merged**